protostar/cairo_testing/cairo_injectable_function_runner.py

```diff
@@ -50,7 +50,8 @@ def run_cairo_function_by_offset(
             if self._cairo_runner_facade.did_panic():
                 panic_data = self._cairo_runner_facade.get_panic_data()
                 panic_data_short_strs = [
-                    short_string_to_str(datum) for datum in panic_data
+                    short_string_to_str(datum) if is_short_string(datum) else None
+                    for datum in panic_data
```

Comment:
Could we reuse this logic in `protostar/cheatable_starknet/controllers/io.py`? It would be more readable imo

Reply:
added